core/src/impl/Kokkos_Default_Graph_Impl.hpp

```diff
@@ -141,7 +141,7 @@ struct GraphImpl : private ExecutionSpaceInstanceStorage<ExecutionSpace> {
     m_has_been_instantiated = true;
   }
 
-  void submit() {
+  void submit(const ExecutionSpace& /* exec */) {
```

Comment:
That right here should be a big fat FIXME

Reply:
Done.